UM-USElab/gradecraft-development
# Save feedback from instructor when a Learning Objective is awarded

### Status
**READY**

### Description
* When awarding a Learning Objective for a student  as an instructor, feedback provided should be saved and displayed to the instructor and the student when the grade is displayed. Currently, the feedback provided is not saved consistently due to the use of the ng-blur event to trigger the update function used to save the feedback.
* Now feedback is saved and displayed and the ng-change event is used to trigger the update function used to save the feedback.

### Related PRs
learning-objectives-not-shown-in-grades-display | Show Learning Objectives in grade display page for instructors (#4344)

### Migrations
NO

### Steps to Test or Reproduce
1. As an instructor, grade a student for an assignment in a course with Learning Objectives enabled and select a Learning Objective Level to award the student and provide feedback.
2. The feedback should be displayed in the grading edit page for the instructor (i.e. /grades/\<id\>/edit) and the grade display page for the student (i.e. /assignments/\<id\>)

### Impacted Areas in Application
* Providing feedback while awarding a Learning Objective as an instructor
* /templates/learning_objectives/grading.html.haml

======================
Closes #4342
